__tests__/app.test.js

```diff
@@ -476,4 +476,60 @@ describe('GET /api/users', () => {
             expect(body.message).toBe('URL NOT FOUND');
           });
       });
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles (topic query)', () => {
+    test('200: responds with articles filtered by the specified topic', () => {
+      return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeInstanceOf(Array);
+          expect(body.articles).toHaveLength(12);
+          body.articles.forEach(article => {
+            expect(article.topic).toBe('mitch');
+          });
+        });
+    });  
+    test('200: responds with all articles when topic query is omitted', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeInstanceOf(Array);
+          expect(body.articles.length).toBeGreaterThan(0);
```

Comment:
This assertion should prove that this endpoint is doing something different to `.get('/api/articles?topic=mitch')`, but it doesn't 👀  It would be ideal to make it more specific, as you did on line 488

Reply:
Also I guess you already have this test, from when you originally implemented /api/articles